app/views/auctions/_auction.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="js-auction-view">
   <div class="auction-body">
     <div class="auction-description">
+      <h2>Skills</h2>
```

Comment:
I think so...wasn't specified in story, but I imagine we will always have some skills for an auction.